src/app/models/user.ts

```
@@ -1,12 +1,14 @@
+import { Person } from './Person';
 
-export class User {
+export class User implements Person {
 
     firstname: string;
```

I wanted to make User inherit from Person because they share the same fields except password. Just makes sense.  Then you could pass a person in for a user.